app/src/main/res/layout-land/admin_pin_activity.xml

```diff
@@ -20,7 +20,7 @@
       android:layout_width="match_parent"
```

Comment:
@ishant904 please delete this file as we don't need it anymore.

Reply:
@rishidyno can we ignore the submit button constraints. The constraints are different in port and land xml. Can you help me out in this